docs/src/pages/components/slider-styled/CustomizedSlider.tsx

```diff
@@ -0,0 +1,219 @@
+import * as React from 'react';
+import { makeStyles, Theme, createStyles } from '@material-ui/core/styles';
+import Slider from '@material-ui/lab/SliderStyled';
+import styled from '@emotion/styled';
+import Typography from '@material-ui/core/Typography';
+import Tooltip from '@material-ui/core/Tooltip';
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    root: {
+      width: `calc(300px + ${theme.spacing(6)})`,
+    },
+    margin: {
+      height: theme.spacing(3),
+    },
+  }),
+);
+
+interface Props {
+  children: React.ReactElement;
+  open: boolean;
+  value: number;
+}
+
+function ValueLabelComponent(props: Props) {
+  const { children, open, value } = props;
+
+  return (
+    <Tooltip open={open} enterTouchDelay={0} placement="top" title={value}>
+      {children}
+    </Tooltip>
+  );
+}
+
+const iOSBoxShadow =
+  '0 3px 1px rgba(0,0,0,0.1),0 4px 8px rgba(0,0,0,0.13),0 0 0 1px rgba(0,0,0,0.02)';
+
+const marks = [
+  {
+    value: 0,
+  },
+  {
+    value: 20,
+  },
+  {
+    value: 37,
+  },
+  {
+    value: 100,
+  },
+];
+
+const IOSSlider = styled(Slider)({
+  color: '#3880ff',
+  height: 2,
+  padding: '15px 0',
+  '& .MuiSlider-thumb': {
```

Comment:
? Or only when using the CSS string syntax not the JavaScript one?

```suggestion

  '& .MuiSlider-thumb': {
```

Reply:
I would say both, will update it!